pom.xml

```diff
@@ -135,6 +137,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>${maven-jar-plugin.version}</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <mainClass>io.strimzi.kafka.bridge.mqtt.Main</mainClass>
```

Comment:
```suggestion
                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
```

Reply:
We don't need the `Main` class specified here. It's passed on the command line in the run script.